docs/docs/started/deploy.md

```diff
@@ -0,0 +1,73 @@
+
+Nebari can be deployed on a bare-metal server using HPC, on a Cloud provider or on a pre-existing kubernetes cluster. Review the options below to discover which option best suits your needs.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="cloud" label="Cloud" default>
+
+The Cloud deployment of Nebari is considered to be the default option. It enables teams to build and maintain a cost effective and scalable compute/data science platform in the Cloud, providing an [Infrastructure as Code](https://en.wikipedia.org/wiki/Infrastructure_as_code) platform that streamlines the deployment of data science infrastructure.
+
+If you are not sure which option to choose, try this one. It is suitable for most use cases, especially if:
+
+- You require scalable infrastructure
+
+- You aim to have a production environment with GitOps enabled by default
+
+The cloud installation is based on Kubernetes, but knowledge of Kubernetes is NOT required nor is in depth knowledge about the specific provider required either. QHub supports [Amazon AWS](https://docs.qhub.dev/en/latest/source/installation/setup.html#amazon-web-services-aws), [DigitalOcean](https://docs.qhub.dev/en/latest/source/installation/setup.html#digital-ocean), [Google GCP](https://docs.qhub.dev/en/latest/source/installation/setup.html#google-cloud-platform), and [Azure](https://docs.qhub.dev/en/latest/source/installation/setup.html#microsoft-azure).
+
+
+For instructions on installing and deploying Nebari on a Clod provider, please visit ou [Cloud providers](/started/cloud-providers) for a list of the supported cloud providers and their respectful installation guides.
+
+</TabItem>
+<TabItem value="hpc" label="HPC">
+
+Nebari HPC is an opinionated open source deployment of jupyterhub based on an HPC jobscheduler. Nebari HPC is a "distribution" of these packages much like Debian and Ubuntu are distributions of Linux. The high level goal of this distribution is to form a cohesive set of tools that enable:
+
+- environment management via conda and conda-store
+
+- monitoring of compute infrastructure and services
+
+- scalable and efficient compute via jupyterlab and dask
+
+- deployment of jupyterhub on prem without requiring deep devops knowledge of the Slurm/HPC and jupyter ecosystem
+
+The Nebari HPC should be your choice if:
+
+- You have highly optimized code that require highly performant infrastructure
+
+- You have existing infrastructure already available
+
+- You expect that your infrastructure will not exceed the existing resources capabilities
+
+For instructions on installing and deploying Nebari HPC, please visit [How to install and setup Nebari HPC on bare metal machines](/how-tos/nebari-hpc).
+
+:::note
+Although it is possible to deploy Nebari HPC on the Cloud, it is not generally recommended due to possible high costs. For more information, check out the base cost section of the docs.
```

Comment:
Need to add link here to the 'base costs' section of the docs.

Reply:
we can open an issue to come back to this when those docs exist.